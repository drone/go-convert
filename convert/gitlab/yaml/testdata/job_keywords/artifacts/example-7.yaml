# https://docs.gitlab.com/ee/ci/yaml/#artifactsreports

rspec:
  stage: test
  script:
    - bundle install
    - rspec --format RspecJunitFormatter --out rspec.xml
  artifacts:
    reports:
      junit: rspec.xml