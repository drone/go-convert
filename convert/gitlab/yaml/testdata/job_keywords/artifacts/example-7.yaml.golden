rspec:
    artifacts:
        reports:
            junit: rspec.xml
    script:
        - bundle install
        - rspec --format RspecJunitFormatter --out rspec.xml
    stage: test
